cf_cell_methods/pcex_helpers.py

```diff
@@ -0,0 +1,29 @@
+# The functions in this file are helpers for the PCEX backend.
+
+from cf_cell_methods import parse
+# Used by the climate explorer backend to filter datasets
+# PCEX cares about what the LAST cell method is, as this records the type
+# of statistical data used the dataset. Generally users selecting a dataset
+# to view are offered the choice of all datasets representing climatological
+# means and ensemble means, for whom the last cell method will be a mean 
+# calculation. 
+# Datasets representing other statistical calculations - percentiles and 
+# standards deviations - are present in the database to be used to provide
+# additional context to the dataset selected by the user. They are handled
+# differently by the PCEX backend.
+# Some example final cell methods:
+#     * time: mean over days (a climatological mean - user can view this)
+#     * time: standard_deviation over days (a climatological SD - used for context)
+#     * models: mean (PCIC12 climatologies - user can view this)
+#     * models: percentile[n] (hydrology percentile data - used for context)
+# Since PCEX treats means over model ensembles and means over time identically, 
+# all the backend needs to know is the last cell method - "mean", "standard_deviation",
+# "percentile", or other / None. 
+def final_operation(cell_method):
+    try:
+        cm = parse(cell_method)
+        method_str = cm[-1].method.name
+        #strip percentile bonus information if present
+        return method_str.split("[")[0]
```

Comment:
You shouldn't have to do this! The _intention_, at least, is that `cm.method` is an object of type [`Method`](https://github.com/pacificclimate/cf-cell-methods/blob/add_final_method/cf_cell_methods/representation.py#L120). `Method` has two attributes, `name` and `params`. So you ought to just be able to compare `cm.method` as is, no splitting required. This probably speaks to deficient documentation, unless there's a bug in the implementation, but [I don't think so](https://github.com/pacificclimate/cf-cell-methods/blob/add_final_method/tests/test_parser.py#L15-L18).